### PR TITLE
Hiero: Add custom scripts menu

### PR DESCRIPTION
**Pull request by friquette**
## Brief description
As for maya and nuke, [scriptmenu ](https://github.com/Colorbleed/scriptsmenu)integration for hiero.

## Description
Scriptsmenu will help you to easily organize your scripts into a customizable menu that users can quickly browse and search.

## Additional info
You have to update scriptmenu with the latest version.

### CODE REVIEW

File: openpype/hosts/hiero/api/launchforhiero.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import logging
+
+from openpype.vendor.python.common.scriptsmenu import scriptsmenu
+from openpype.vendor.python.common.scriptsmenu.vendor.Qt import QtWidgets
```
Comment (iLLiCiTiT) [inline openpype/hosts/hiero/api/launchforhiero.py:4]: You don't have to do so explicit imports on vendorized modules.
from scriptsmenu import scriptsmenu
from Qt import QtWidgets
```

File: openpype/hosts/hiero/api/menu.py [hunk 2]
```diff
@@ -138,3 +140,33 @@ def menu_install():
...
+        return
+
+    # run the launcher for Maya menu
+    studio_menu = launchforhiero.main(title=_menu.title())
+
+    # apply configuration
+    studio_menu.build_from_configuration(studio_menu, config)
+
+
+add_scripts_menu()
```
Comment (iLLiCiTiT) [inline openpype/hosts/hiero/api/menu.py:172]: Never call functions in global scope. This should be moved to function `install` in `pipeline.py`.

**Review by jakubjezek001 — CHANGES_REQUESTED** [review]

File: openpype/settings/defaults/project_settings/hiero.json [hunk 3]
```diff
@@ -1,4 +1,32 @@
...
+        "workfile": "hrox",
+        "yetiRig": "hrox"
+    },
+    "hiero-dirmap": {
+        "enabled": false,
+        "paths": {
+            "source-path": [],
+            "destination-path": []
+        }
+    },
```
Comment (iLLiCiTiT) [inline openpype/settings/defaults/project_settings/hiero.json:16]: There are not schemas for these settings. Also you've created defaults manually, please use settings UI in develop mode to create default settings.
```suggestion
```

### How to run settings UI in develop mode
There is a script `openpype/tools/run_settings`. If you run it you should have available `Modify defaults` checkbox. From that moment if you save settings they will change default settings in json file.
image

**Review by iLLiCiTiT — APPROVED** [review]
Looks good. @jakubjezek001 please check and merge.

**Review by jakubjezek001 — APPROVED** [review]
Awesome work, thanx!

**Pull request merged**